.github/ISSUE_TEMPLATE.md

```diff
@@ -12,6 +12,11 @@
 
 **What is the [`ember-cli-rails-addon`][addon] version (from `package.json`)?**
 
+**What is your application server (e.g. `puma`, `unicorn`, `thin`, `webrick`,
+etc.)?**
+
+**Is your application server multi-threaded or multi-process?**
```

Comment:
How about one question, perhaps something like:

> Is your application server multi-threaded (such as `puma` and `unicorn`) or is it multi-process (such as `thin` and `webrick`)?

(It could be that I still don't understand if this is the right question to ask.)